Packs/DefaultPlaybook/ReleaseNotes/2_0_6.md

```diff
@@ -0,0 +1,9 @@
+
+#### Playbooks
+
+##### Default
+- Fixed a bug where the playbook was submitting invalid data for URL detonation, causing errors in certain cases.
+- Fixed a bug where the playbook was using the older Detonate URL - Generic playbook, which, when VirusTotal was enabled, would result in an error due to usage of a removed command.
```

Comment:
based on what I saw in this PR , you didn't touch that section at all

Reply:
You're correct, it seems the Default playbook was not updated on that server earlier which caused a confusion. I will remove it